### PR TITLE
Form styling

### PR DESCRIPTION
**Pull request by ryanlalchand**

### CODE REVIEW

**Comment by JoeDuncko** [comment]
Note that this diff seems to include diffs that is in another PR. That might cause some issues with squash and merge.

File: pages/jobs-form.tsx [hunk 1]
```diff
@@ -8,45 +8,122 @@ export default function JobsForm() {
...
+          <br />
+          <br />
         </div>
         <div>
           <label htmlFor="listingLink">
             Listing link or contact email (optional): &nbsp;
-          </label>
-          <input type="text" id="listingLink" name="listingLink" />
+          </label>{" "}
+          <br />
```
Comment (JoeDuncko) [inline pages/jobs-form.tsx:108]: `<br>`s are not a great way to add spacing. Use styles instead.

File: pages/jobs-form.tsx [hunk 2]
```diff
@@ -8,45 +8,122 @@ export default function JobsForm() {
...
+          />{" "}
+          <br />
+          <br />
         </div>
         <div>
-          <label htmlFor="email">Your Email: &nbsp;</label>
-          <input type="email" id="email" name="email" required />
+          <label htmlFor="email">Your Email: &nbsp;</label> <br />
+          <input
+            className="bg-CYgray
```
Comment (JoeDuncko) [inline pages/jobs-form.tsx:34]: Might want to create a `FormField` component that abstracts some of props.

File: pages/jobs-form.tsx [hunk 3]
```diff
@@ -8,45 +8,122 @@ export default function JobsForm() {
...
+            name="name"
+            required
+          />{" "}
+          <br />
+          <br />
         </div>
         <div>
-          <label htmlFor="email">Your Email: &nbsp;</label>
-          <input type="email" id="email" name="email" required />
+          <label htmlFor="email">Your Email: &nbsp;</label> <br />
```
Comment (JoeDuncko) [inline pages/jobs-form.tsx:32]: `&nbsp;` is not a great way to add spacing, use styles instead.

File: pages/jobs.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ export default function Jobs() {
       <h1 className="text-3xl p-2">Jobs</h1>
       <h2 className="text-xl p-2">Community-sourced job opportunities</h2>
       <Link href="/jobs-form">
-        <a>Add a job listing</a>
+        <a className="hover:text-gray-300">Add a job listing</a>
```
Comment (JoeDuncko) [inline pages/jobs.tsx:13]: Maybe "Submit a Job"?

**Review by JoeDuncko — CHANGES_REQUESTED** [review]
Good start! Check out the comments

**Review by garrett-huggins — APPROVED** [review]

**Review by JoeDuncko — APPROVED** [review]

**Pull request merged**